kfs-core/src/main/java/edu/arizona/kfs/fp/document/validation/impl/DisbursementVoucherAccountingLineAccessibleValidation.java

```diff
@@ -0,0 +1,158 @@
+package edu.arizona.kfs.fp.document.validation.impl;
```

Comment:
I don't believe these changes are needed or related to UAF-2155. The bug reported in KATTS-22 was fixed by the Foundation in KFS 5.0.x. See https://jira.kuali.org/browse/KFSOLD-22667.